docs/developer-guide/syntax-reference.rst

```diff
@@ -839,7 +839,9 @@ In the above statements from_item is one of the following:
 -  ``from_item LEFT JOIN from_item ON join_condition``
 
 The WHERE clause can refer to any column defined for a stream or table,
-including the two implicit columns ``ROWTIME`` and ``ROWKEY``.
+including the two implicit columns ``ROWTIME`` and ``ROWKEY``. If those
+columns contain Strings or Scalar values, the BETWEEN keyword can be used
```

Comment:
```suggestion
columns contain strings or scalar values, you can use the BETWEEN keyword
```

Reply:
Passive tense is a bad habit of mine! Thanks for checking it :)